src/style.css

```diff
@@ -0,0 +1,140 @@
+@font-face {
+  font-family: Avenir_heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: avenir_book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}
```

Comment:
Follow the checklist, please 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/99403198/0c17b689-6bc6-4932-ae9f-e9dd3180f23a)